x/mongo/driver/operation.go

```diff
@@ -1838,6 +1838,7 @@ func (op Operation) publishFinishedEvent(ctx context.Context, info finishedInfor
 				DriverConnectionID: info.driverConnectionID,
 				Message:            logger.CommandSucceeded,
 				Name:               info.cmdName,
+				DatabaseName:       op.Database,
```

Comment:
This adds DatabaseName to the CommandSucceeded logger. This is not a requirement in the logging specification: https://github.com/qingyang-hu/specifications/blob/master/source/command-logging-and-monitoring/command-logging-and-monitoring.rst#command-succeeded-message

Does it need to be added?

Reply:
I've updated the spec at: https://github.com/mongodb/specifications/commit/6b267ddc7cca303a722d2893106fad4810709f06#diff-81d41cc6fd279ee033885bc989eea1823c252f4b3e0c00a8605e03f1290ef864R498

Reply:
The portion of the API you've updated is for event monitoring, not logging. The logging API fields are here: https://github.com/qingyang-hu/specifications/blob/master/source/command-logging-and-monitoring/command-logging-and-monitoring.rst#command-succeeded-message

The common logging fields are here: https://github.com/qingyang-hu/specifications/blob/master/source/command-logging-and-monitoring/command-logging-and-monitoring.rst#common-fields

Reply:
I see my mistake, thank you for clarifying offline!